README.md

```diff
@@ -26,6 +25,8 @@ version of the program gets to start again from `main`.
 
 ## How do I use it?
 
+([See below if you'd rather build it yourself from source.](#building-it-yourself))
```

Comment:
Can we work this into the text a bit better? Also I don't think I would include the . in the link.

Maybe:

Unless you want to [build Jaguar from source](...), start by downloading ...